Set up

adamzerner opened the pull request:
It took me long enough, but I think I've got the "set up" for the server completed. Meaning:

1. Beginnings of the file structure and routing.
2. CRUD for Posts
3. CRUD for Users

ajcrites commented:
Add your email address on the machine you are pushing from to match one of the email addresses on your Github account so Github knows it's you that is making these commits.

adamzerner commented:
> Add your email address on the machine you are pushing from
I assume you're talking about git config. If so, I just added it. I can't believe it wasn't there already - I even remember me and John giving that presentation on git config during On Ramp! Anyway, I added it. If that's not what you're talking about, please let me know.
